models/marts/core/int_all_graph_resources.sql

```diff
@@ -96,7 +96,8 @@ joined as (
         unioned_with_calc.loaded_at_field, 
         unioned_with_calc.loader, 
         unioned_with_calc.identifier,
-        unioned_with_calc.hard_coded_references -- NULL for non-model resources
+        unioned_with_calc.hard_coded_references, -- NULL for non-model resources
+        unioned_with_calc.is_excluded -- NULL for metrics and exposures
```

Comment:
any reason not to extend the logic to metrics and exposures too?

Reply:
i am perfectly satisfied with that! just wanted to make sure we'd thought about it